cmd/rhelemeter-server/main_e2e_test.go

```diff
@@ -16,13 +16,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-kit/kit/log"
+	"github.com/go-kit/log"
 	"github.com/gogo/protobuf/proto"
 	"github.com/golang/snappy"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/expfmt"
 	"github.com/prometheus/prometheus/prompb"
-	"github.com/prometheus/prometheus/util/testutil"
+	"github.com/stretchr/testify/require"
```

Comment:
Hmm can we keep the `testutil` notation here and replace with: https://github.com/efficientgo/core/tree/main/testutil? This is what we use throughout Thanos as well, so would make this repo much more consistent.